Maple2.Server.Game/Commands/PlayerCommand.cs

```diff
@@ -164,4 +164,46 @@ private class InfoCommand : Command {
             ctx.Console.Out.WriteLine($"  Rotation: {session.Player.Rotation}");
         }
     }
+
+    private class DebugAnimationCommand : Command {
+        private readonly GameSession session;
+
+        public DebugAnimationCommand(GameSession session) : base("debuganims", "Prints player animation info.") {
+            this.session = session;
+
+            var enable = new Argument<bool?>("enable", () => true, "Enables & disables debug messages. Prints all animation state if true.");
+
+            AddArgument(enable);
+
+            this.SetHandler<InvocationContext, bool?>(Handle, enable);
+        }
+
+        private void Handle(InvocationContext ctx, bool? enabled) {
+            session.Player.AnimationState.DebugPrintAnimations = enabled ?? true;
+
+            string message = enabled ?? true ? "Enabled" : "Disabled";
+            ctx.Console.Out.WriteLine($"{message} animation debug info printing");
+        }
+    }
+
+    private class DebugSkillsCommand : Command {
+        private readonly GameSession session;
+
+        public DebugSkillsCommand(GameSession session) : base("debugskills", "Prints player skill packet info.") {
+            this.session = session;
+
+            var enable = new Argument<bool?>("enable", () => true, "Enables & disables debug messages. Prints all skill cast packets if true.");
+
+            AddArgument(enable);
+
+            this.SetHandler<InvocationContext, bool?>(Handle, enable);
+        }
+
+        private void Handle(InvocationContext ctx, bool? enabled) {
+            session.Player.DebugSkills = enabled ?? true;
+
+            string message = enabled ?? true ? "Enabled" : "Disabled";
+            ctx.Console.Out.WriteLine($"{message} skill cast packet debug info printing");
+        }
+    }
```

Comment:
Can we have these in DebugCommands too?

Reply:
Whoops, did I forget to remove em from Player? I did bring em over to debug, but probably forgot to remove em here